.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -1,36 +1,36 @@
 name: "Feature"
 description: "Suggest an idea for the Oakcan app."
-labels: ["feature-request"]
+labels: [ "feature-request" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)